includes/orders/classes/class-refund-validator.php

```diff
@@ -345,7 +345,8 @@ private function validate_item_and_add_totals( $original_item, $amounts_to_refun
 			}
 
 			// This is our fallback.
-			$attempted_amount = isset( $original_item->{$column_name} ) ? $original_item->{$column_name} : 0.00;
+			$attempted_amount = isset( $original_item->{$column_name} ) ? floatval( $original_item->{$column_name} ) : 0.00;
+			$maximum_amount   = floatval( $maximum_refundable_amounts[ $column_name ] );
```

Comment:
Can you revert this? Just because `edd_sanitize_amount()` will already do it and I don't want to confuse ourselves over which one we actually need.